### PR TITLE
Add a picow_blink_fast_clock example

### PR DESCRIPTION
**Pull request by peterharperuk**
We have picow_blink_slow_clock, so we probably need an example of running it faster?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,7 @@ App|Description
 [picow_access_point](pico_w/wifi/access_point) | Starts a WiFi access point, and fields DHCP requests.
 [picow_blink](pico_w/wifi/blink) | Blinks the on-board LED (which is connected via the WiFi chip).
 [picow_blink_slow_clock](pico_w/wifi/blink_slow_clock) | Blinks the on-board LED (which is connected via the WiFi chip) with a slower system clock to show how to reconfigure communication with the WiFi chip under those circumstances
+[picow_blink_fast_clock](pico_w/wifi/blink_slow_clock) | Blinks the on-board LED (which is connected via the WiFi chip) with a faster system clock to show how to reconfigure communication with the WiFi chip under those circumstances
```
Comment (kilograham) [inline README.md:174]: um fast

File: pico_w/wifi/blink/CMakeLists.txt [hunk 2]
```diff
@@ -31,3 +31,24 @@ pico_add_extra_outputs(picow_blink_slow_clock)
 
 # add url via pico_set_program_url
 example_auto_set_url(picow_blink_slow_clock)
+
+# This version should behave exactly the same, but it runs the sys clock slowly.
```
Comment (kilograham) [inline pico_w/wifi/blink/CMakeLists.txt:35]: um fast

File: pico_w/wifi/blink/CMakeLists.txt [hunk 3]
```diff
@@ -31,3 +31,24 @@ pico_add_extra_outputs(picow_blink_slow_clock)
...
+        )
+target_link_libraries(picow_blink_fast_clock
+        pico_stdlib              # for core functionality
+        pico_cyw43_arch_none     # we need Wifi to access the GPIO, but we don't need anything else
+        hardware_clocks
+        )
+# This requires us to modify the pio divisor to successfully communicate with the cyw43 chip
+target_compile_definitions(picow_blink_fast_clock PRIVATE
+        CYW43_PIO_CLOCK_DIV_INT=4
+        CYW43_PIO_CLOCK_DIV_FRAC=0
```
Comment (kilograham) [inline pico_w/wifi/blink/CMakeLists.txt:47]: probably use FRAC8 or whatever the correct new define is

Comment (lurch) [reply pico_w/wifi/blink/CMakeLists.txt:47]: Yeah, a `git grep` says that `CYW43_PIO_CLOCK_DIV_FRAC8` is correct:
```
src/rp2_common/pico_cyw43_driver/include/pico/cyw43_driver.h:// PICO_CONFIG: CYW43_PIO_CLOCK_DIV_FRAC8, Fractional part of the clock divider for communication with the wireless chip 0-255, type=int, min=0, max=255, default=0, group=pico_cyw43_driver
```

**Review by kilograham — APPROVED** [review]

**Pull request merged**